app/components/Vision/index.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import styles from './Vision.css';
+import cx from 'classnames';
+
+const Vision = ({ title, summary, paragraphs, left }) => {
+  return (
+    <div>
+      <h3
+        className={cx(styles.vision__title, left && styles.vision__titleLeft)}
```

Comment:
Don't think we practice double underscores anywhere else in the app? Using css modules, it should not be necessary with unique names since you import styles locally anyways – so I guess you are better off just doing `styles.title`, alternatively visionTitle. 🙂Might be an idea to reuse or unify the titles styles, so that titles across the pages are not different – unless specifically wanted here!